classes/security/authorization/internal/SubmissionFileBaseAccessPolicy.inc.php

```diff
@@ -19,19 +19,19 @@ class SubmissionFileBaseAccessPolicy extends AuthorizationPolicy {
 	/** @var PKPRequest */
 	var $_request;
 
-	/** @var string File id and revision, separated with a dash (e.g. 15-1) */
-	var $_fileIdAndRevision;
+	/** @var int Submission file id */
+	var $_submissionFileId;
 
 	/**
 	 * Constructor
 	 * @param $request PKPRequest
-	 * @param $fileIdAndRevision string If passed, this policy will try to
+	 * @param $submissionFileId int If passed, this policy will try to
 	 * get the submission file from this data.
 	 */
-	function __construct($request, $fileIdAndRevision = null) {
```

Comment:
(Also very happy to see these weird glued-together-IDs removed. They were always a nasty wrinkle.)